docs/pages/web/chartgraph.js

```diff
@@ -0,0 +1,599 @@
+          `}
+        >
+          <MainSection.Card
+            sandpackExample={<SandpackExample code={precision} name="Precision" />}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          title="Colors"
+          description={`Colors on data series are set automatically for best contrast.
+
+If different graphs need to be compared simultaneously, see example below, color in time series can be set in the \`elements\` prop setting each color \`color='01'\` individually.
+        `}
+        >
+          <CombinationNew cardSize="xs" color={['01', '02', '03', '04', '05', '06']}>
+            {({ color }) => (
+              <Box
+                dangerouslySetInlineStyle={{
+                  __style: {
+                    backgroundColor: `var(--g-colorDataVisualization${color})`,
+                  },
+                }}
+                height={100}
+                width={100}
+              />
+            )}
+          </CombinationNew>
+          <CombinationNew cardSize="xs" color={['07', '08', '09', '10', '11', '12']}>
+            {({ color }) => (
+              <Box
+                dangerouslySetInlineStyle={{
+                  __style: {
+                    backgroundColor: `var(--g-colorDataVisualization${color})`,
+                  },
+                }}
+                height={100}
+                width={100}
+              />
+            )}
+          </CombinationNew>
+          <MainSection.Card
+            cardSize="lg"
+            sandpackExample={<SandpackExample code={colors} name="Colors" layout="column" />}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          title="Layout"
+          description={`ChartGraph supports 4 layouts. Legend positions automatically on each layout for easier comprehension.
+
+Single axis:
+1. vertical (default)
+2. horizontal
+
+Dual axis:
+3. verticalBiaxial
+4. horizontalBiaxial
+
+Requires props: \`layout\`
+
+        `}
+        >
+          <MainSection.Card sandpackExample={<SandpackExample code={layout} name="Layout" />} />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          title="Range"
+          description={`ChartGraph automatically sets the minimum and maximum axis values. The \`range\` prop allows  adjusting them in case we need broader range values in the axis.
+
+Requires props: \`range\`
+
+        `}
+        >
+          <MainSection.Card sandpackExample={<SandpackExample code={range} name="Range" />} />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          title="Responsive"
+          description={`ChartGraph is a responsive component. ChartGraph's width adjust to the parent container. In order to render properly in requires a parent container with set dimensions.
+
+
+          For vertical layour, ChartGraph has a set height based on the amount of ticks (five or three). For horizontal layouts, ChartGraph has a set height.
```

Comment:
Typo here and sentence issues.

This isn't intentional right:
![Uploading image.png…]()